column-div/column-div.lua

```diff
@@ -0,0 +1,155 @@
+--[[
+column-div - leverage Pandoc native divs to make balanced and unbalanced column
+             and other things based on class name and attributes.
+
+Copyright:  © 2021 Christophe Agathon <christophe.agathon@gmail.com>
+License:    MIT – see LICENSE file for details
+
+Credits:    Romain Lesur and Yihui Xie for the original column filter
+            implementation (output in beamer format).
+
+Output:     latex, pdf, html
+
+Usage:                          classname   attributes
+            balanced columns    .columns    column-count
+            columns(container)  .columns
+            column(each column) .column     width(percent) valign(t|c|b)
+            other divs          .<somename> data-latex
+
+            See README.md for details
+
+Note:       You need to include multicol latex package to get balanced columns
+            in latex or pdf
+            I tried to use well known html or latex parameter.
+            Even if lua doen't like hyphens like in column-count.
+
+Bugs:       * html rendering throws a warning [WARNING] Ignoring duplicate attribute style="width:60%;".
+            when widht AND color are set and totally ignore the width
```

Comment:
```suggestion
            when width AND color are set and totally ignore the width
```

Lets figure out whether this bug is in this code or not. It should be pretty easy to setup an experiment to see if plain Pandoc with no filters has this problem, no?

Reply:
1) Plain Pandoc is OK
2) Pandoc + filter **seems to be** OK if the div uses only a `width` parameter (the filter **do no** process the `width` parameter)
3) Pandoc + filter **not** OK if there is a `width` parameter **and** at least a second parameter.

Two options so far :
1) I do something wrong in the code
2) I trigger a tricky bug

Need some time to figure out since it is my first Pandoc filter ever.

Reply:
My best guess for now is that Pandoc doesn't check that a filter already set the `style` attribute so it attempts to create a new one with the value "width:xx%;".
I added the processing of the width attribute in the filter in addition to the other attributes. When the filter has nothing to do, it lets Pandoc process the width.